test/PurchaseArrayTest.java

```diff
@@ -0,0 +1,32 @@
+package test;
+
+import module.*;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PurchaseArrayTest {

```

Comment:
unit testing is your weak side. 

take the course [Java: Testing with JUnit on the learn platform](https://learn.epam.com/detailsPage?id=a2f13cb6-109d-4632-951d-d074a4ab95af).

other functionality must be tested. 
basically you've tested some jdk features but not your code.